fitbenchmarking/fitting_benchmarking.py

```diff
@@ -210,7 +213,7 @@ def do_fitting_benchmark_one_problem(prob, group_results_dir, minimizers, use_er
     return results_fit_problem
 
 
-def make_plots(prob, group_results_dir, best_fit, wks, previous_name, count, user_func):
+def make_plots(prob, visuals_dir, best_fit, wks, previous_name, count, user_func):
```

Comment:
Is visuals_dir the support pages dir?

Reply:
Yes.

Reply:
also where you change function name I would expect this name to change in documentation. If no nothing documented for old variable please add documentation